manifests/rhoai/telemetry/servicemonitor.yaml

```
@@ -0,0 +1,48 @@
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: training-operator
+  endpoints:
+  - port: https-metrics
+    path: /metrics
+    scheme: HTTPS
+    tlsConfig: {}
+    interval: 30s
+    metricRelabelings:
+      # Keep only the primary telemetry metric
+      - action: keep
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)